frontend/src/Onboarding/GenericOnboarding.tsx

```diff
@@ -57,14 +112,45 @@ const theme = createMuiTheme({
   },
 });
 
-const steps = ["", ""];
+const steps = ["", "", ""];
+
+interface SelectableCourseProps {
+  readonly onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  readonly courseText: string;
+}
 
-interface Props {
+interface GenericOnboardingTemplateProps {
   screen: number;
   children: any;
 }
 
-export const GenericOnboardingTemplate: React.FC<Props> = ({
+interface OnboardingSelectionTemplateProps {
+  readonly screen: number;
+  readonly mainTitleText: string;
+  readonly onSubmit: () => void;
+  readonly to: string;
+}
+
+const SelectableCourse: React.FC<SelectableCourseProps> = ({
```

Comment:
Abstraction for displaying a single selectable course on the transferable exams or completed courses screens